LayoutTests/http/wpt/identity/idl.https-expected.txt

```diff
@@ -0,0 +1,88 @@
+PASS Navigator includes NavigatorOnLine: member names are unique
+PASS Navigator includes NavigatorContentUtils: member names are unique
+PASS Navigator includes NavigatorCookies: member names are unique
+PASS Navigator includes NavigatorPlugins: member names are unique
+PASS Navigator includes NavigatorConcurrentHardware: member names are unique
+PASS Credential interface: existence and properties of interface object
+PASS Credential interface object length
+PASS Credential interface object name
+PASS Credential interface: existence and properties of interface prototype object
+PASS Credential interface: existence and properties of interface prototype object's "constructor" property
+PASS Credential interface: existence and properties of interface prototype object's @@unscopables property
+PASS Credential interface: attribute id
+PASS Credential interface: attribute type
+PASS CredentialsContainer interface: existence and properties of interface object
+PASS CredentialsContainer interface object length
+PASS CredentialsContainer interface object name
+PASS CredentialsContainer interface: existence and properties of interface prototype object
+PASS CredentialsContainer interface: existence and properties of interface prototype object's "constructor" property
+PASS CredentialsContainer interface: existence and properties of interface prototype object's @@unscopables property
+PASS CredentialsContainer interface: operation get(optional CredentialRequestOptions)
+PASS CredentialsContainer interface: operation store(Credential)
+PASS CredentialsContainer interface: operation create(optional CredentialCreationOptions)
+PASS CredentialsContainer interface: operation preventSilentAccess()
+PASS CredentialsContainer must be primary interface of navigator.identity
+PASS Stringification of navigator.identity
+PASS CredentialsContainer interface: navigator.identity must inherit property "get(optional CredentialRequestOptions)" with the proper type
+PASS CredentialsContainer interface: calling get(optional CredentialRequestOptions) on navigator.identity with too few arguments must throw TypeError
+PASS CredentialsContainer interface: navigator.identity must inherit property "store(Credential)" with the proper type
+PASS CredentialsContainer interface: calling store(Credential) on navigator.identity with too few arguments must throw TypeError
+PASS CredentialsContainer interface: navigator.identity must inherit property "create(optional CredentialCreationOptions)" with the proper type
+PASS CredentialsContainer interface: calling create(optional CredentialCreationOptions) on navigator.identity with too few arguments must throw TypeError
+PASS CredentialsContainer interface: navigator.identity must inherit property "preventSilentAccess()" with the proper type
+FAIL PasswordCredential interface: existence and properties of interface object assert_own_property: self does not have own property "PasswordCredential" expected property "PasswordCredential" missing
```

Comment:
These FAILs are expected, as we don't implement PasswordCredential or FederatedCredential.